.travis.yml

```diff
@@ -0,0 +1,35 @@
+# [minor]  Build and push 1.1.0 image
+# [major]  Make the version tag to 2.0.0
+sudo: true
```

Comment:
[Travis are now recommending removing the __sudo__ tag](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration).